main.js

```diff
@@ -0,0 +1,74 @@
+const choices = ["rock","paper","scissors"]
+let playerScore = 0;
+let computerScore = 0;
+
+function playerChoice (){
+    let input = prompt("Choose rock, paper, or scissors");
+    while (input == null){  
+        input = prompt("Choose rock, paper, or scissors");
+    }
+    input = input.toLowerCase();
+    let check = validateInput(input);
+    while (check == false){
+        input = prompt("Please choose rock, paper, or scissors");
+             while (input == null) {
+                 input = prompt("Choose rock, paper, or scissors");
+             }
```

Comment:
Will help you rewrite this function, but for now let's focus on the rest of these revisions 😃

Reply:
still need your help with this. :)